torchrl/objectives/iql.py

```diff
@@ -423,7 +420,7 @@ def _loss_qvalue(self, tensordict: TensorDictBase) -> Tuple[Tensor, Tensor]:
         target_value = self.value_estimator.value_estimate(
             tensordict, target_params=self.target_value_network_params
         ).squeeze(-1)
-        tensordict_expand = vmap(self.qvalue_network, (None, 0))(
+        tensordict_expand = self._vmap_qvalue_networkN0(
```

Comment:
what are these N0 or 00 standing for? not super intuitive for me

Reply:
None, 0 and 0, 0
because we have 2 different vmaps
other names are welcome! `none_0`?